sdk/servicebus/azure-servicebus/swagger/servicebus-swagger.json

```diff
@@ -0,0 +1,757 @@
+          }
+        },
+        "RequiresDuplicateDetection": {
+          "description": "A value indicating if this queue requires duplicate detection.",
+          "type": "boolean",
+          "xml": {
+            "namespace": "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect"
+          }
+        },
+        "RequiresSession": {
+          "description": "A value that indicates whether the queue supports the concept of sessions.",
+          "type": "boolean",
+          "xml": {
+            "namespace": "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect"
+          }
+        },
+        "Status": {
+          "$ref": "#/definitions/EntityStatus"
+        },
+        "SupportOrdering": {
+          "description": "A value that indicates whether the queue supports ordering.",
+          "type": "boolean",
+          "xml": {
+            "namespace": "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect"
+          }
+        }
+      }
+    },
+    "QueueRuntimeInfo":{
+      "description": "Service Bus queue metrics.",
+      "type": "object",
+      "xml": {
+        "namespace": "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect"
+      },
+      "properties": {
+        "QueueName": {
+          "description": "Name of the queue",
+          "type": "string"
+        },
+        "AccessedAt": {
+          "description": "Last time a message was sent, or the last time there was a receive request to this queue.",
+          "type": "string",
+          "format": "date-time",
+          "xml": {
+            "namespace": "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect"
+          }
+        },
+        "CreatedAt": {
+          "description": "The exact time the queue was created.",
+          "type": "string",
+          "format": "date-time",
+          "xml": {
+            "namespace": "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect"
+          }
+        },
+        "UpdatedAt": {
+          "description": "The exact time a message was updated in the queue.",
+          "type": "string",
+          "format": "date-time",
+          "xml": {
+            "namespace": "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect"
+          }
+        },
+        "SizeInBytes": {
+          "description": "The size of the queue, in bytes.",
+          "type": "integer",
+          "xml": {
+            "namespace": "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect"
+          }
+        },
+        "MessageCount": {
+          "description": "The number of messages in the queue.",
+          "type": "integer",
+          "xml": {
+            "namespace": "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect"
+          }
+        },
+        "MessageCountDetails": {
+          "$ref": "#/definitions/MessageCountDetails",
+          "xml": {
+            "name": "CountDetails",
```

Comment:
Don't mix ref and xml (see previous comment)

Reply:
fixed